mm2src/mm2_main/src/lp_ordermatch/simple_market_maker.rs

```diff
@@ -518,7 +518,11 @@ async fn update_single_order(
     let resp = update_maker_order(ctx, req)
         .await
         .map_to_mm(OrderProcessingError::OrderUpdateError)?;
-    info!("Successfully update order for {} - uuid: {}", key_trade_pair, resp.uuid);
+    info!(
+        "Successfully update order for {key_trade_pair} - uuid: {} - rate: ({:.4} {key_trade_pair})",
+        resp.uuid,
+        calculated_price.to_decimal()
```

Comment:
I believe `MmNumber` implements `Display` trait. It's even converted into `BigDecimal` on `Display::fmt` call :)
Please correct me if I'm wrong

Reply:
yes correct but doesn't round up even with `{:.4}`..

output example `0.4739357839020400000000`

Reply:
Got it, thanks!